style.css

```diff
@@ -0,0 +1,124 @@
+head {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.4;
+  color: white;
+  margin: 0;
+}
+
+body::before {
+  content: '';
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: -1;
+  background: var(--color-darkblue);
+  background-image: linear-gradient(115deg, rgba(58, 58, 158, 0.8), rgba(136, 136, 206, 0.7));
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
- [ ] As you can see, you don't have any background image. It would be great if you add the recommended background image.

![Screenshot 2023-01-17 004945](https://user-images.githubusercontent.com/75938662/212748104-402b99ff-75b2-4648-85a6-4da1d9a54ec2.jpg)